### PR TITLE
add community section

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Description
In the handbook, we have covered many topics in detail, but there is limited information on how we interact with the community. I believe we should add a separate community section in the handbook.

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by ZJvandeWeg** [comment]
Deferring my review to @Hasmin-AC to free up some of my day.

File: src/handbook/community/community-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+navTitle: Community Guidelines
+---
+
+## Node-RED is FlowFuse
+
+We strongly encourage the belief that there is no difference between the Node-RED and FlowFuse communities. Whether you are a FlowFuse community member or an employee, you should treat both community members equally. Remember, the success of FlowFuse lies in the success of Node-RED.
```
Comment (gdziuba) [inline src/handbook/community/community-guidelines.md:7]:
```suggestion
We strongly encourage the belief that the Node-RED and FlowFuse communities are the same.Whether you are a FlowFuse community member or an employee, you should treat both community members equally. Remember, the success of FlowFuse lies in the success of Node-RED.
```

File: src/handbook/community/community-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### General Guidelines
+
+We provide the following guidelines to help ensure that our community remains respectful, collaborative, and supportive. These principles are grounded in common sense and shared values. By adhering to them, we can create a welcoming environment for all members and foster a culture of mutual respect and growth.
```
Comment (gdziuba) [inline src/handbook/community/community-guidelines.md:11]:
```suggestion
We provide the following guidelines to help ensure that our community remains respectful, collaborative, and supportive. These principles are grounded in common sense and shared values. Adhering to them can create a welcoming environment for all members and foster a culture of mutual respect and growth.
```

File: src/handbook/community/event-and-webinars.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+navTitle: Events and Webinars
+---
+
+## FlowFuse Webinars and Events
+
+At FlowFuse, we are committed to delivering value to our customers and users. That's why we conduct monthly webinars and other initiatives to enhance experience and knowledge.
```
Comment (gdziuba) [inline src/handbook/community/event-and-webinars.md:7]:
```suggestion
At FlowFuse, we are committed to delivering value to our customers and users. To enhance their experience and knowledge, we conduct monthly webinars and other initiatives.

```

File: src/handbook/community/forums-and-support.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- **Tutorials and Guides**: Detailed tutorials and guides to help you get started and master advanced topics.
+  - [FlowFuse Tutorials and Guides](/blog/)
+
+- **Ebook**: The ultimate beginner guide to professional Node-RED.
+  - [The Ultimate Beginner Guide to a Professional Node-RED](/ebooks/beginner-guide-to-a-professional-nodered/)
+
+- **Webinars**: Live webinars where experts share their knowledge, provide updates on FlowFuse, and offer the opportunity for community members to ask questions directly during the webinar.
+  - [FlowFuse Webinars](./event-and-webinars#monthly-webinars)
+
+- **Community Discord**: A channel for users to discuss FlowFuse and Node-RED, share resources, and ask questions.
```
Comment (gdziuba) [inline src/handbook/community/forums-and-support.md:43]:
```suggestion
- **Community Discord**: This is a channel for users to discuss FlowFuse and Node-RED, share resources, and ask questions.
```

File: src/handbook/community/forums-and-support.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  - [Discord Channel](https://discord.gg/2RrvW8dkrF)
+
+- **Ticket Support and Instant Chat**: Personalized and real-time support through our help center.
+  - [FlowFuse Help Center](/support/)
+
+- **GitHub Issues**: A platform for submitting issues for new features or bug reports. Provide detailed information to help our developers understand and address your issue.
```
Comment (gdziuba) [inline src/handbook/community/forums-and-support.md:49]:
```suggestion
- **GitHub Issues**: This is a platform for submitting issues for new features or bug reports. Provide detailed information to help our developers understand and address your issue.
```

**Review by gdziuba — APPROVED** [review]
Looks good.  Accept the suggestion at your discretion.

**Comment by sumitshinde-84** [comment]
Thanks, @gdziuba, for the review!

File: src/handbook/community/community-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+navTitle: Community Guidelines
+---
+
+## Node-RED is FlowFuse
+
+We strongly encourage the belief that the Node-RED and FlowFuse communities are the same.Whether you are a FlowFuse community member or an employee, you should treat both community members equally. Remember, the success of FlowFuse lies in the success of Node-RED.
```
Comment (ZJvandeWeg) [inline src/handbook/community/community-guidelines.md:7]:
```suggestion
We strongly encourage the belief that the Node-RED and FlowFuse communities are the same. Whether you are a FlowFuse community member or an employee, you should treat both community members equally. Remember, the success of FlowFuse lies in the success of Node-RED.
```

File: src/handbook/community/community-guidelines.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (ZJvandeWeg) [inline src/handbook/community/community-guidelines.md:1]: Can we remove duplication between: https://flowfuse.com/handbook/company/communication/#node-red-community-interactions.

I think that section I just linked should be removed and the contents, where not yet done, to be present in these pages.

Comment (ZJvandeWeg) [reply src/handbook/community/community-guidelines.md:1]: As outreach is a marketing activity, I think this should all be under `/handbook/customer/community/*`.

File: src/handbook/community/community-guidelines.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+## Node-RED is FlowFuse
+
+We strongly encourage the belief that the Node-RED and FlowFuse communities are the same.Whether you are a FlowFuse community member or an employee, you should treat both community members equally. Remember, the success of FlowFuse lies in the success of Node-RED.
+
+### General Guidelines
+
+We provide the following guidelines to help ensure that our community remains respectful, collaborative, and supportive. These principles are grounded in common sense and shared values. Adhering to them can create a welcoming environment for all members and foster a culture of mutual respect and growth.
+
```
Comment (ZJvandeWeg) [inline src/handbook/community/community-guidelines.md:12]: Link to the CoC of Node-RED: https://nodered.org/about/conduct/?

**Comment by ZJvandeWeg** [comment]
Thanks a lot for the PR @sumitshinde-84 !

File: src/handbook/community/community-guidelines.md (hunk 7 above)
Comment (Hasmin-AC) [reply src/handbook/community/community-guidelines.md:1]: +1 @sumitshinde-84 can you please move it.

**Pull request merged**